app.py

```diff
@@ -10,10 +10,12 @@
 import tempfile
 import time
 from collections import namedtuple
-from dataclasses import asdict, dataclass
-from enum import Enum, unique
-from typing import List, Tuple
+from dataclasses import asdict, dataclass, is_dataclass
+from enum import Enum, IntEnum, unique
+from functools import partial, wraps
+from typing import Any, List, Tuple
 
+import config
```

Comment:
I am wondering why your local pre-commit or other linter keeps moving this line?

Reply:
I think maybe there is a little conflict between `isort` and `black`, this line may have been added to staging before I added the black rule for `isort`, but now it's normal, sorry for that👻

Reply:
seems it's isort's issus, we should clearly tell it the source of some modules/packages, see [issue](https://github.com/PyCQA/isort/issues/457) and this [comment](https://github.com/PyCQA/isort/issues/457#issuecomment-304263970) for more discuss, i'll fix it.